src/webview/js/js.js

```diff
@@ -926,9 +943,27 @@ const handleLongPress = (target: Element) => {
     return;
   }
 
+  // Prettier bug on nested ternary
+  /* prettier-ignore */
+  const targetType = target.matches('.header')
+    ? 'header'
+    : target.matches('a')
+      ? 'link'
+      : 'message';
+  const messageNode = target.closest('.message');
+
+  if (
+    targetType === 'message'
+    && messageNode
```

Comment:
This is still a bit messy, but it's only extending a way that the existing logic in this function is already messy, and I think it's fine for now.

When I merge this, I may try splitting the conditionals in this function up to follow the style of the click handler above:
* first, see if target matches `.header`, if so act on that and return.
* then, see if matches `a`, if so act on that and return.
* then, see if muted message, if so act on that and return.
* finally, act on default case.

because I think that'll make it somewhat clearer, and also the difference in clarity will grow rapidly if we add any more cases here.